consultation_analyser/consultations/migrations/0073_remove_responseannotationtheme_unique_theme_assignment_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 5.2.5 on 2025-09-10 14:59
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+def remove_human_assigned_themes(apps, schema_editor):
+    ResponseAnnotationTheme = apps.get_model("consultations", "ResponseAnnotationTheme")
+    ResponseAnnotationTheme.objects.exclude(assigned_by__isnull=True).delete()
```

Comment:
here i am deleting all historic human assigned themes so that the theme assignment is unique - is this right?